README.md

```diff
@@ -8,68 +8,58 @@ API to manage [JDK Flight Recordings](https://openjdk.java.net/jeps/328).
+- `go` v1.15
+- [`operator-sdk`](https://github.com/operator-framework/operator-sdk) v1.3.0
 - [`cert-manager`](https://github.com/jetstack/cert-manager) v1.1.0 (Recommended)
-- `podman`
+- `podman` or `docker`
 - `ginkgo` (Optional)
 
 ## Instructions
-Operator SDK requires the `GOROOT` environment variable to be set to the root
-of your Go installation. In Bash, this can be done with
-`export GOROOT="$(go env GOROOT)"`.
+`make generate manifests manager` will trigger code/YAML generation and compile
+the operator controller manager, along with running some code quality checks.
 
-`make image` will create an OCI image to the local registry, tagged as
+`make oci-build` will build an OCI image from the generated YAML and compiled
+binary to the local registry, tagged as
 `quay.io/rh-jmc-team/container-jfr-operator`. This tag can be overridden by
-setting the environment variable `IMAGE_TAG`.
+setting the environment variables `IMAGE_NAMESPACE` and `OPERATOR_NAME`.
+`IMAGE_VERSION` can also be set to override the tagged version.
 
-To create a CSV bundle, use `CSV_VERSION=1.2.3 make csv`. This will generate
-a CSV and CRDs for a bundle versioned 1.2.3 at
-`./deploy/olm-catalog/container-jfr-operator-bundle/1.2.3`.
-
-To (re)build the Operator bundle for packaging in catalogs (ex. OperatorHub),
-`CSV_VERSION=1.2.3 make bundle`. This will create an Operator bundle image
-containing the required YAML manifest files and metadata for publishing the
-bundle, based on the manifest files at
-`./deploy/olm-catalog/container-jfr-operator-bundle/1.2.3`. Push the resulting
-bundle image to an image registry such as quay.io.
-
-To create an index of these bundles, use `make index`. Push the index to an
-image registry such as quay.io.
+To create an OLM bundle, use `make bundle`. This will generate a CSV, CRDs and
+other manifests, and other required configurations for an OLM bundle versioned
+with version `$IMAGE_VERSION` in the `bundle/` directory. `make bundle-build`
+will create an OCI image of this bundle, which can then be pushed to an image
+repository such as `quay.io`.
 
 # Setup / Deployment
-## UI-Guided Deployment
+## Bundle Deployment
 
-The operator can be deployed using the Operator Marketplace in the graphical
-OpenShift console by first adding a custom catalog source. A YAML definition
-of such a custom source is at `deploy/olm-catalog/catalog-source.yaml`, which
-can be added to your cluster by
-`oc create -f deploy/olm-catalog/catalog-source.yaml` or simply `make catalog`.
-This method allows the latest released version of the operator to be installed
-into the cluster with the same method and in the same form that end users would
-receive it.
+The operator can be deployed using OLM using `make deploy_bundle`. This will
+deploy `quay.io/rh-jmc-team/container-jfr-operator-bundle:$IMAGE_VERSION` to
+your configured cluster using `oc`. You can set the variables `IMAGE_NAMESPACE`
```

Comment:
This should probably be `kubectl` to be consistent with the above text.

Reply:
This ends up calling `operator-sdk run bundle`, and I'm not sure if this does call `oc`/`kubectl` or what it does exactly, so I've updated the README to be a little less specific here and cover oc/kubectl/kubeconfig...